### DISCUSSION TITLE
Do I need to include the tracking script on all pages?

### DISCUSSION THREAD

**Discussion by larnsce** [plausible/analytics#725] [Anything Else]
Hi Everyone

I have just setup plausible for my domain lse.de and included the tracking script in the index.html. I really like it so far!

On the Dashboard under Top Pages, I do not see when someone opens a page on my website (e.g. lse.de/blog.html).

Do I need to include the tracking script on every page that I want to track? Would the dashboard still summaries all page view for the domain lse.de?

Thanks for your help.
Lars

**Comment by Vigasaurus** [comment]
You do indeed need to put the tracking script on every html file - most people do this with a build or deploy-time tool or one of the many automatic integrations available. https://plausible.io/docs/integration-guides 

Plausible will still track everything perfectly fine along with properly tracking user sessions even if they do go through multiple html files with all separate instances of the script, assuming you put the same `data-domain` on all pages.
